cfme/common/__init__.py

```diff
@@ -238,16 +241,37 @@ def step(self, *args, **kwargs):
         self.prerequisite_view.toolbar.policy.item_select('Manage Policies')
 
 
+class AssignedTags(ParametrizedView):
+    PARAMETERS = ("tag",)
+    ALL_TAGS = ".//a[contains(@class, 'pf-remove-button')]"
+    tag = Text(ParametrizedLocator(
+        ".//div[@class='category-label']/parent::li/following-sibling::"
+        "li/descendant::a[contains(@class, 'pf-remove-button')]")
+    )
+
+    def remove(self):
+        self.tag.click()
```

Comment:
If `tag` is a parametrized Text locator, then this would need a parameter passed?

Reply:
Same as above. 
It's very similar to `@ParametrizedView.nested` in `class AlertEmail`